attenuation.py

```diff
@@ -0,0 +1,513 @@
+import os
+import numpy as np
+from scipy import interpolate
+from unyt import Angstrom, unyt_quantity, unyt_array
+
+from dust_extinction.grain_models import WD01
+
+from synthesizer import exceptions
+
+
+this_dir, this_filename = os.path.split(__file__)
+
+__all__ = ["PowerLaw", "MW_N18", "Calzetti2000", "GrainsWD01", "LMC_M23"]
+
+def N09_tau(lam, slope, cent_lam, ampl, gamma):
```

Comment:
This is hugely pedantic so I apologise, but please use snake_case for functions and variables and reserve CamelCase for class names. (This does need fixing in a bunch of places elsewhere in the code but I think I'm getting to the end of the list)